themes/default/content/docs/intro/console/accounts.md

```diff
@@ -74,7 +73,23 @@ When you run [`pulumi login`]({{< relref "/docs/reference/cli/pulumi_login" >}})
 
 ![Access tokens](/images/docs/reference/service/access-tokens.png)
 
-You can create additional access tokens on this tab by selecting **Create token**. Similarly, you may remove an unused token by highlighting it on the list and selecting "Remove access token".
+### Creating Access Tokens
+
+To create an access token:
+
+1. Select your profile picture.
+1. Select **Settings**.
+1. Navigate to **Access Tokens**.
+1. Select **Create token**.
```

Comment:
Should token always be capitalized or is the variation intentional?

Reply:
these match up with the ui, right now our nav items and headings use title case, and buttons are sentence case. we should update our nav items at some point to use sentence case too, then this wont look as funky here.